fastlane/spec/actions_specs/import_certificate_spec.rb

```diff
@@ -66,7 +66,7 @@
         password = 'testpassword'
 
         keychain_path = File.expand_path(File.join('~', 'Library', 'Keychains', keychain))
-        expected_command = "security import #{cert_name} -k '#{keychain_path}' -P #{password} -T /usr/bin/codesign -T /usr/bin/security"
+        expected_command = "security import #{cert_name} -k #{keychain_path} -P #{password} -T /usr/bin/codesign -T /usr/bin/security"
```

Comment:
Same here:
isn't this missing `.shellescape` then?

Reply:
Same here:
> Escaping is a no-op here because there are no special characters in the test parameters. 
> Good catch and the other parameters should be escaped too.